qutip/tests/test_orbital.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from scipy.special import sph_harm
+from qutip.orbital import orbital
+import qutip
+import numpy as np
+
+
+def test_orbital_single_ket():
+    """ Checks output for a single ket as input"""
+    # setup mesh for theta and phi
+    theta_list = np.linspace(0, np.pi, num=50)
+    phi_list = np.linspace(0, 2 * np.pi, num=100)
+    for theta, phi in zip(theta_list, phi_list):
+        # set l and m
+        for l in range(0, 5):
+            for m in range(-l, l + 1):
+                q = qutip.basis(2 * l + 1, l + m)
+                # check that outputs are the same,
+                # note that theta and phi are interchanged for scipy
+                assert sph_harm(m, l, phi, theta) == orbital(theta, phi, q)
+
+
+def test_orbital_multiple_ket():
+    """ Checks if the combination of multiple kets works """
+    theta_list = np.linspace(0, np.pi, num=50)
+    phi_list = np.linspace(0, 2 * np.pi, num=100)
+    l, m = 5, 2
+    q1 = qutip.basis(2 * l + 1, l + m)
+    l, m = 3, -1
+    q2 = qutip.basis(2 * l + 1, l + m)
+    for theta, phi in zip(theta_list, phi_list):
+        assert orbital(theta, phi, q1, q2) == sph_harm(2, 5, phi,
+                                                       theta) + sph_harm(-1, 3,
+                                                                         phi,
+                                                                         theta)
```

Comment:
This does not look clean.
Computing `expected` on another line would look better.